client/client-multi/client-cpp/kaa/KaaClient.hpp

```diff
@@ -124,8 +126,10 @@ class KaaClient : public IKaaClient,
     void checkClientState(State expected, const std::string& message);
     void checkClientStateNot(State unexpected, const std::string& message);
 
-private:
     State                                           clientState_ = State::CREATED;
+    LoggerPtr        logger_;
```

Comment:
Surrounding code has different members placement and formatting. Please, fix this.